pom.xml

```diff
@@ -12,8 +12,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
-        <vaadin.version>17.0-SNAPSHOT</vaadin.version>
-        <flow.version>3.2-SNAPSHOT</flow.version>
+        <vaadin.version>18.0.0.alpha1</vaadin.version>
```

Comment:
are v18 code/doc available somewhere.
I'd like to play with the new form api and lit form elements.
Thanks !

Reply:
Hi, really glad to hear that you are interested in the new form API, it was already available in Vaadin 17. You can find the documentation on [vaadin.com](https://vaadin.com/docs/v17/flow/client-side-forms/tutorial-binder.html). The whole **Creating Client-side Forms** is about the new form API.

Reply:
Thanks a lot for your quick answer (and sorry to comment on this closed CL).

I was wondering as I can see some docs but I was not seeing `@vaadin/form` in the v17 branch.

I'll dig a little more.

Reply:
Hey Haijian,

I can not find the form package (`@vaadin/form`) in [`vaadin/vaadin`](https://github.com/vaadin/vaadin).

I checked the master branch which is supoosed to be v18.

It's probably a stupid mistake but your help would be appreciated,
Thanks

Reply:
Hi, you can find the source code for `@vaadin/form` [here](https://github.com/vaadin/flow/tree/master/flow-client/src/main/resources/META-INF/resources/frontend/form). It's included inside Vaadin, not intended to be used separately, because one important feature is the Java to TypeScript model auto code generation, which depends on the server-side code.